File: 백준/Bronze/10810. 공 넣기/공 넣기.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

let [N, M] = input
  .shift()
  .split(' ')
  .map((v) => Number(v));
let ball = input.map((v) => v.split(' ').map((v) => Number(v)));
let basket = new Array(N).fill(0);

for (a = 0; a < ball.length; a++) {
  let [i, j, k] = ball[a];
  for (b = i - 1; b <= j - 1; b++) {
    basket[b] = k;
  }
}

console.log(basket.join(' '));

File: 백준/Bronze/10810. 공 넣기/공 넣기.py
n, m = list(map(int, input().split(' ')))
arr = []
for x in range(0, n):
    arr.append(0)

for y in range(0, m):
    i, j, k = list(map(int, input().split(' ')))
    for z in range(i-1, j):
        arr[z] = k

print(" ".join(list(map(str, arr))))